cdc/kv/region_state.go

```diff
@@ -31,33 +31,56 @@ const (
 	stateRemoved uint32 = 2
 )
 
-type singleRegionInfo struct {
-	verID  tikv.RegionVerID
+type regionInfo struct {
+	verID tikv.RegionVerID
+	// The span of the region.
+	// Note: The span is not always the real span of the region, it is the span of the
```

Comment:
this comment is not easy to understand.

Reply:
Please add comments to explain that this situation is possible only when a single region contains data from multiple tables.